### DISCUSSION TITLE
v0.1.0-preview5.2

### DISCUSSION THREAD

**Discussion by rabbitism** [irihitech/Semi.Avalonia#101] [Announcements]
Native AOT Demo is available!

## What's Changed
* Fix margin values in Palette demo by @zdpcdt in https://github.com/irihitech/Semi.Avalonia/pull/89
* Input remake by @rabbitism in https://github.com/irihitech/Semi.Avalonia/pull/92
* Add NativeAOT support to Semi.Avalonia and Demo.  by @rabbitism in https://github.com/irihitech/Semi.Avalonia/pull/98
* Fix FlyoutPresenter and ButtonSpinner style by @rabbitism in https://github.com/irihitech/Semi.Avalonia/pull/100

**Full Changelog**: https://github.com/irihitech/Semi.Avalonia/compare/v0.1.0-preview5.1...v0.1.0-preview5.2

Packages are available via nuget
```bash
dotnet add package Semi.Avalonia --version 0.1.0-preview5.2
dotnet add package Semi.Avalonia.ColorPicker --version 0.1.0-preview5.2
dotnet add package Semi.Avalonia.DataGrid --version 0.1.0-preview5.2
```

<hr /><em>This discussion was created from the release <a href='https://github.com/irihitech/Semi.Avalonia/releases/tag/v0.1.0-preview5.2'>v0.1.0-preview5.2</a>.</em>

**Comment by qqmaplealone** [comment]
Nice
